addons/purchase/models/purchase.py

```diff
@@ -625,12 +634,12 @@ def retrieve_dashboard(self):
                    WHERE m.create_date >= %s
                      AND m.model = 'purchase.order'
                      AND m.message_type = 'notification'
-                     AND v.old_value_char = 'RFQ'
-                     AND v.new_value_char = 'RFQ Sent'
+                     AND v.old_value_char IN %s
+                     AND v.new_value_char IN %s
```

Comment:
Như này có được không?
```suggestion
                     AND v.old_value_char IN = _( 'RFQ')
                     AND v.new_value_char IN = _('RFQ Sent')
```

Reply:
> Như này có được không?

@davidtranhp em nghĩ như này không được anh ạ, vì nếu dùng như này khi ở tiếng việt thì chỉ so sánh với tên 'Y/c báo giá' anh ạ còn 'RFQ' sẽ sót
cc @royleviindoo

Reply:
Bổ sung dịch cho 2 cái này chắc phải được chứ nhỉ?

Reply:
Em thử test giúp anh cái này nhé: https://github.com/odoo/odoo/pull/87426

Anh code chay, chưa test.

Reply:
@davidtranhp em thử và không được anh ạ, mình truyền _('RFQ') thì sẽ chỉ lấy giá trị dịch theo ngôn ngữ người dùng hiện tại thôi anh ạ. Em gửi video bên dưới

https://user-images.githubusercontent.com/41574005/160545657-7e658116-1642-4776-847a-8b7c6319f478.mp4

Reply:
cc @royleviindoo

Reply:
Chú bổ sung dịch tiếng Việt cho PR đó chưa?

Reply:
Ví dụ chỗ
```
#. module: purchase
#: model:ir.model.fields.selection,name:purchase.selection__purchase_order__state__draft
#, python-format
msgid "RFQ"
msgstr "Y/c báo giá"
```
phải thay bằng 
```
#. module: purchase
#: code:addons/purchase/models/purchase.py:0
#: model:ir.model.fields.selection,name:purchase.selection__purchase_order__state__draft
#, python-format
msgid "RFQ"
msgstr "Y/c báo giá"
```

Reply:
> Chú bổ sung dịch tiếng Việt cho PR đó chưa?

@davidtranhp em lấy nhánh của anh bên trên về test ạ

Reply:
để em kiểm tra lại ạ

Reply:
Vấn đề gặp phải do đa ngôn ngữ anh ạ
Ví dụ
PO1: User 1 dùng tiếng Việt để đổi giai đoạn thì tracking sẽ lưu giá trị `Yêu cầu báo giá`
PO2: User 2 dùng tiếng Anh để đổi giai đoạn thì tracking sẽ lưu giá trị `RFQ`
Nhưng khi User 3 đang dùng tiếng Anh, thì cách code của anh sẽ chỉ lấy được PO2, còn PO1 thì không lấy được anh ạ

Reply:
@davidtranhp em thử rồi không được anh ạ, _('RFQ') chỉ cho ra giá trị theo ngôn ngữ người dùng hiện tại ạ nên query kia sẽ không hết được bản ghi của mail_tracking_value ạ

Reply:
OK